### PR TITLE
1509 add become a partner page

### PR DESCRIPTION
**Pull request by solazio**
## Done
- Add `/partners/become-a-partner` page

## QA
- Go to [/partners/become-a-partner](http://0.0.0.0:8002/partners/become-a-partner) 
- Compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1509)

## Issue

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1509

### CODE REVIEW

File: templates/partners/become-a-partner.html [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+              <option value="2">Bionic Beaver</option>
+              <option value="3">Xenial Xerus</option>
+            </select>
+            <textarea id="partbershipText" rows="5" placeholder="Please tell us more about the opportunity you see in partnering with Canonical..."></textarea>
+            <input type="checkbox" id="CanonicalUpdatesOptIn">
+            <label for="CanonicalUpdatesOptIn"><small>I agree to receive information about Canonical’s products and services.</small></label>
+            <p>
+              <small>In submitting this form, I confirm that I have read and agree to <a href="https://ubuntu.com/legal/data-privacy/newsletter">Canonical’s Privacy Notice</a> and <a href="https://ubuntu.com/legal/data-privacy">Privacy Policy</a>.</small>
+              </p>
+            <button type="submit">Apply now</button>
```
Comment (anthonydillon) [inline templates/partners/become-a-partner.html]: Positive button please

File: templates/partners/become-a-partner.html [hunk 2]
```diff
@@ -0,0 +1,479 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <section class="p-strip--image is-dark">
+    <div class="p-strip--suru-background">
+      <div class="u-fixed-width">
```
Comment (anthonydillon) [inline templates/partners/become-a-partner.html]: We should wrap this content in a row and col

File: templates/partners/become-a-partner.html [hunk 3]
```diff
@@ -0,0 +1,484 @@
...
+{% block content %}
+  <section class="p-strip--image is-dark">
+    <div class="p-strip--suru-top">
+      <div class="row">
+        <div class="col-12">
+          <h1>Become a partner</h1>
+          <p class="u-sv3">Partnering with Ubuntu and Canonical consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+        </div>
+      </div>
+      <div class="row u-hide--small">
```
Comment (anthonydillon) [inline templates/partners/become-a-partner.html]: Seems odd to hide all of this content on small screens

File: templates/partners/become-a-partner.html [hunk 4]
```diff
@@ -0,0 +1,484 @@
...
+    </div>
+  </section>
+  <section class="p-strip--image is-dark">
+    <div class="p-strip--suru-bottom">
+      <div class="row">
+        <div class="col-8">
+          <h1>Find a partner</h1>
+          <p class="u-sv3">Canonical’s network of Ubuntu partners spans the full range of technology activities. We offer partner programmes for software vendors, hardware OEMs, mobile carriers, channel partners, retailers and public cloud providers.</p>
+        </div>
+        <div class="col-4 u-align--center u-vertically-center">
```
Comment (anthonydillon) [inline templates/partners/become-a-partner.html]: I think this image could do with being hidden on small screens

Comment (solazio) [reply templates/partners/become-a-partner.html]: @anthonydillon makes sens. I will hide it.

File: templates/partners/become-a-partner.html (hunk 3 above)
Comment (solazio) [reply templates/partners/become-a-partner.html]: @anthonydillon ok. I will keep the images on mobile.

**Comment by anasereijo** [comment]
- [ ] according to the design, the left side text ("How we work with our partners", etc) is 5 cols and the form is 6 cols. So this means we should have an empty col between them.

- [ ] form: could we have more padding? the elements are too close to the edge of the card (background)

Canonical _ The company behind Ubuntu (1)

**Comment by anasereijo** [comment]
looks good!

**Pull request merged**